### PR TITLE
Add presidio SDK

### PR DESCRIPTION
**Pull request by navalev**
This PR is built on top of pr #1 and #2

- add presidio SDK
- call presidio APIs
- unit tests (using mock API handler)

### CODE REVIEW

**Review by SharonHart — APPROVED** [review]

File: src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Api/AnonymizerApiMock.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Presidio.Api;
+using Presidio.Client;
+using Presidio.Model;
+
+namespace Fhir.Anonymizer.Core.UnitTests.Api
+{
+        public class AnonymizerApiMock : IAnonymizerApi
```
Comment (itye-msft) [inline src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Api/AnonymizerApiMock.cs:11]: Why is there a need to test the SDK?
We should assume the SDK is tested.

Comment (itye-msft) [reply src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Api/AnonymizerApiMock.cs:11]: Same for the analyzer API.
We should only test here logic we added to FHIR Tools

File: src/Microsoft.Health.Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
     {"path": "FamilyMemberHistory.deceased", "method": "redact"},
     {"path": "FamilyMemberHistory.condition.onset as Age", "method": "perturb", "span": 0.1, "rangeType": "proportional", "roundTo": 0},
     {"path": "FamilyMemberHistory.condition.onset", "method": "redact"},
-    {"path": "ClinicalImpression.description", "method": "redact"},  
+    {"path": "ClinicalImpression.description", "method": "presidio"},
```
Comment (itye-msft) [inline src/Microsoft.Health.Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json:65]: Ok, for now.
I think in the future we might want to generalize this.
for example: "method": "unstructured-data-replace"
etc, and use a configuration to imply which "unstructured-data" anonymizer to use.

**Review by itye-msft — CHANGES_REQUESTED** [review]

File: src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Api/AnonymizerApiMock.cs (hunk 1 above)
Comment (SharonHart) [reply src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Api/AnonymizerApiMock.cs:11]: The purpose of AnonymizerApiMock is to test The ApiHandler, need to pass an instance of IAnonymizerApi that's not calling the actual service

File: src/Microsoft.Health.Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json (hunk 2 above)
Comment (SharonHart) [reply src/Microsoft.Health.Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json:65]: makes sense

**Review by itye-msft — APPROVED** [review]

**Pull request merged**